src/resources/postcss/components/full/_ical-link.pcss

```diff
@@ -53,3 +53,17 @@
 		}
 	 }
 }
+
+/* -----------------------------------------------------------------------------
+ * Theme Overrides - Twenty Twenty One
+ * ----------------------------------------------------------------------------- */
+.tribe-theme-twentytwentyone {
+	.tribe-events {
+		.tribe-events-c-ical__link {
+			&:focus:not(.wp-block-button__link):not(.wp-block-file__button) {
+				color: var(--color-background);
+				background-color: var(--color-accent-primary);
+			}
+		}
+	}
+}
```

Comment:
let's move this into the `.tribe-events` block, much like the Twenty Seventeen override above